decidim-core/lib/decidim/coauthorable.rb

```diff
@@ -82,6 +82,14 @@ def creator
         coauthorships.order(:created_at).first
       end
 
+      # Checks whether the user is the creator of the given proposal
+      #
+      # author - the author to check for authorship
+      def created_by?(author)
+        return false if author.nil?
+        author.id == creator_author.id
```

Comment:
Compare objects instead of ids please

```suggestion
        author == creator_author
```